src/PrestaShopBundle/Entity/Repository/RequestSqlRepository.php

```diff
@@ -83,10 +83,19 @@ public function getCount()
     }
 
     /**
-     * {@inheritdoc}
+     * Get query that searches grid rows.
+     *
+     * @param SearchCriteriaInterface $searchCriteria
+     *
+     * @deprecated deprecated since 1.7.8.0
+     * @see RequestSqlQueryBuilder::getSearchQueryBuilder
+     *
+     * @return QueryBuilder
      */
     public function getSearchQueryBuilder(SearchCriteriaInterface $searchCriteria = null)
     {
+        @trigger_error(sprintf('The "%s()" method is deprecated since 1.7.8.0', __METHOD__), E_USER_DEPRECATED);
```

Comment:
```suggestion
        @trigger_error(
            sprintf(
                'The "%s()" method is deprecated since 1.7.8.0. Use %s instead.', 
                __METHOD__, 
                RequestSqlQueryBuilder::class . '::getSearchQueryBuilder()'
            ), 
            E_USER_DEPRECATED
        );
```